src/main/java/no/nav/foreldrepenger/mottak/oppslag/pdl/PDLConnection.java

```diff
@@ -76,7 +76,13 @@ public class PDLConnection implements PingEndpointAware {
     }
 
     public Person hentSøker() {
-        return hentSøkerInternal(b -> b.erNyligFødt(cfg.getBarnFødtInnen()) && !b.erNyligDød(cfg.getDødSjekk()));
+        return hentSøkerInternal(b -> {
+            var nyligFødt = b.erNyligFødt(cfg.getBarnFødtInnen());
+            if (!nyligFødt) {
+                return false;
+            }
+            return b.getDødsfall() == null || b.getDødsfall().isEmpty() || b.erNyligDød(cfg.getDødSjekk());
```

Comment:
Litt rart å ha den logikken der og ikke i barn metoden? Ikke veldig lett å lese.

Lage en metode som returnere boolean og legg inn som første filter for barn metoden (kodelinje 126)?